source/drivers/ST7735.cpp

```diff
@@ -65,7 +65,7 @@
 namespace codal
 {
 
-ST7735::ST7735(ScreenIO &io, Pin &cs, Pin &dc) : io(io), cs(cs), dc(dc), work(NULL)
+ST7735::ST7735(ScreenIO &io, Pin &cs, Pin &dc) : io(io), cs(&cs), dc(&dc), work(NULL)
```

Comment:
I think at this point, cs and dc should just be pointers... references are only useful for mandatory parameters.

Reply:
Default parameters as null?

Up to you on both points.

Reply:
yeah, but I would have to change stuff in pxt then...